nano/sign.go

```diff
@@ -0,0 +1,83 @@
+package nano
+
+import (
+	"bytes"
+	"crypto/sha512"
+
+	"github.com/pkg/errors"
+
+	crypto "github.com/tendermint/go-crypto"
+)
+
+const (
+	App       = 0x80
+	Init      = 0x00
+	Update    = 0x01
+	Digest    = 0x02
+	MaxChunk  = 253
+	KeyLength = 32
+	SigLength = 64
+)
+
+var separator = []byte{0, 0xCA, 0xFE, 0}
```

Comment:
fine for now, but we should eventually remove this separator in favor of just having hardcoded slice offsets.

Reply:
(just a heads-up)